src/main/resources/config/application-artemis.yml

```diff
@@ -88,3 +88,5 @@ artemis:
         feedback-consistency-url: http://localhost:8001/feedback_consistency
         base64-secret: YWVuaXF1YWRpNWNlaXJpNmFlbTZkb283dXphaVF1b29oM3J1MWNoYWlyNHRoZWUzb2huZ2FpM211bGVlM0VpcAo=
         token-validity-in-seconds: 10800
+    resultListener:
```

Comment:
Why is this needed?

The problem was that during tests a lot of tests failed as after making a REST call the mock mvc removes the authentication object meaning no changing sql statements (create, update, delete) are possible any more. Thus the entity listener needs to be deactivated during tests.